unit/util/range.cpp

```diff
@@ -245,6 +245,39 @@ SCENARIO("range tests", "[core][util][range]")
       }
     }
   }
+  GIVEN("Two vectors of different sizes.")
+  {
+    const std::vector<int> int_vector{814, 51};
+    const std::vector<std::string> string_vector{"foo", "bar", "baz", "bay"};
+    WHEN("We zip the vectors")
+    {
+      auto range = make_range(int_vector).zip<false>(string_vector);
```

Comment:
:pick: Please repeat this test, but with `int_vector` and `string_vector` swapped.